src/App.tsx

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 import './App.css';
-import Header from './header';
-import Content from './content';
+//import Header from './components/organisms/header';
+//import Content from './content';
 import {BrowserRouter as Router} from 'react-router-dom';
+import Main from './components/pages/main';
+import List from './components/pages/list';
 
 
 const App: React.FC = () => {
   return (
     <Router>
```

Comment:
ここは、最終的には以下の様になると思います。
現状は、仮で実装していると思いますが、レイアウト崩れや現状動いている挙動に問題が発生する様な変更は控えましょう。

今回はこのままでOKとします。

```suggestion
    <div>
      <Route>
    </div>
```

ちなみに、上の例で定義している `<Route>` は、自作のルーティング用コンポーネントで、BrouserRouterではないよ。

Reply:
レビューいただきましてありがとうございます
次回作業よりご指摘いただきました箇所を考慮しながら作業してまいります